### ISSUE TITLE
Downloads links for binaries are dead

### ISSUE THREAD

**Issue by lrq3000** [Tox-Client/client#127]
Example for Windows 64 bits: https://downloads.tox.chat/windows/x64

Does not work for me, website inaccessible.

**Comment by mva1985** [comment]
try...

https://tox.chat/download.html

**Comment by lrq3000** [comment]
Ok, same result: https://downloads.utox.io/stable/uTox_win64.exe  --> ERR_CONNECTION_REFUSED

Also the utox_runner updater quits and says that there is an "error fetching the latest version data. Please check your internet connection. Exiting now..."

**Comment by mva1985** [comment]
looks like you my have to build it yourself

https://github.com/uTox/uTox/releases

**Comment by mva1985** [comment]
I successfully downloaded qtox from

https://tox.chat/download.html

**Comment by lrq3000** [comment]
Yes qTox works, but I wanted to try uTox because it has a wider cross-platform coverage, which is preferable for my friends to switch over. I hope uTox binaries download will be restored :-/
